packages/engine/src/utils/index.coffee

```diff
@@ -4,6 +4,7 @@ module.exports =
   buffer: require './buffer'
   diff: require './diff'
   error: require './error'
+  ini: require './ini'
```

Comment:
This is shall be in a separate commit, you are asking me to review something which is not related to testall